Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/values.yaml

```diff
@@ -1,28 +1,32 @@
-# Default values for azure policy for kubernetes
+# Default values for Azure Policy Add-on on Azure Arc for Kubernetes clusters
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-## Microsoft Azure Policy Add-on in Kubernetes
+## Microsoft Azure Policy Add-on on Azure Arc for Kubernetes clusters
 azurepolicy:
   image:
-    name: "emmcmill/azure-policy-kubernetes"
-    tag: "ccs-test"
-## Resource ID for aks-engine is the id of the resource group
+    name: "mcr.microsoft.com/azure-policy/policy-kubernetes-addon-prod"
+    tag: "prod_20200224.1"
+## Replace <your_resource_id> with your Azure Arc for Kubernetes cluster resource Id. For example: /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/testRG/Microsoft.Kubernetes/connectedClusters/testCluster
+## Replace <your_client_id> with app Id of the service principal created for Azure Policy Add-on to communicate to Azure Policy service.
+## Replace <your_client_secret> with password of the service principal created for Azure Policy Add-on to communicate to Azure Policy service.
+## Replace <your_tenant_id> with tenant of the service principal created for Azure Policy Add-on to communicate to Azure Policy service.
   env:
     resourceid: <your_resource_id>
     clientid: <your_client_id>
     clientsecret: <your_client_secret>
     tenantid: <your_tenant_id>
 
-
-## Configure resource requests and limits
-## ref: http://kubernetes.io/docs/user-guide/compute-resources/
-##
+## OPA Gatekeeper for Kubernetes clusters
+gatekeeper:
+  image:
+    name: "mcr.microsoft.com/oss/open-policy-agent/gatekeeper"
+    tag: "v3.1.0-beta.7"
+  args:
+    auditIntervalSeconds: 60
+    constraintViolationsLimit: 20
   resources:
-    deployment:
-      requests:
-        cpu: 30m
-        memory: 50Mi
-      limits:
-        cpu: 100m
-        memory: 200Mi
+    cpuLimit: 1000m
```

Comment:
Why are gatekeeper limits injectable while add-on is hardcoded?

Reply:
Because of Gatekeeper OPA cache. Following the AKS YAML patterns where these are customizable.

Reply:
Sweet as